core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/Benchmark.scala

```diff
@@ -104,6 +104,7 @@ class Benchmark(
     val gcTimes = ArrayBuffer[Long]()
     //For tracking maximum GC over iterations
     for (i <- 0 until minIters) {
+      System.gc()  // ensures GC for a consistent state across different iterations
```

Comment:
If we are triggering GC before every iteration, should we remove the GC collection defined at the beginning of this method?

Reply:
It does not matter that much.